pnpm-workspaces/apps/alpha/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../tsconfig.json"
```

Comment:
Is it worth having these in each folder? I think less config is better for this example..?

Reply:
Probably not, I can have just the one in `alpha` and convert `beta` to pure js.

Reply:
I think we can just have one tsconfig in the root and the libs+apps can all share that?

👍 for converting one of the apps to pure js just to show the absolute minimum in one of them

Reply:
I might need some help solving that. Will that require `export_files`? So that the root tsconfig is available for alpha? Or is defining a `ts_config` in the root sufficient and setting the `tsconfig` field on `ts_project` to `//:tsconfig`? @jbedard

Reply:
Looks like you got it 👍